src/algorithms/fardetectors/MatrixTransferStatic.cc

```diff
@@ -204,9 +206,9 @@ void eicrecon::MatrixTransferStatic::process(
     gpos = gpos/dd4hep::mm;
     pos0 = pos0/dd4hep::mm;
 
-	std::cout << "gpos.z() = " << gpos.z() << " pos0.z() = " << pos0.z() << std::endl;
+	//std::cout << "gpos.z() = " << gpos.z() << " pos0.z() = " << pos0.z() << std::endl;
 
-	std::cout << "[gpos.x(), gpos.y()] = " << gpos.x() <<", "<< gpos.y() << "  and [pos0.x(), pos0.y()] = "<< pos0.x()<< ", " << pos0.y() << std::endl;
+	//std::cout << "[gpos.x(), gpos.y()] = " << gpos.x() <<", "<< gpos.y() << "  and [pos0.x(), pos0.y()] = "<< pos0.x()<< ", " << pos0.y() << std::endl;
```

Comment:
```suggestion
```

Reply:
I cleaned up all comments, except for one pair of cout statements that I will literally need in less than one week to work on the new geometry for the beam pipe and subsequent matrix re-tuning. I will delete them with the PR related to that topic, they aren't meant for long-term debugging and this will make it easier for me to remember to remove them next week.